config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openshift/cluster-nfd-operator
```

Comment:
actually those values should be:
newName: quay.io/openshift/origin-cluster-nfd-operator
newTag: "4.19"

Reply:
that value was set in my commit by mistake